app/assets/stylesheets/application.scss

```diff
@@ -4,11 +4,17 @@
 @import "images";
 @import "buttons";
 @import "teams/*";
+@import "users/*";
 
 body {
-  font-family: 'Roboto', sans-serif;
+  font-family: "Helvetica Neue",Helvetica,Arial,sans-serif;
+  background-color: #EEEEEE;
```

Comment:
Color literals like `#EEEEEE` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#EEEEEE` should be written as `#eeeeee`